<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditClientWindow</class>
 <widget class="QMainWindow" name="EditClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="editClientPushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>600</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Редагувати клієнта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Редагування клієнта&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="maleRadioButton">
       <property name="text">
        <string>Чоловіча стать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="femaleRadioButton">
       <property name="text">
        <string>Жіноча стать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>611</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lastname">
       <property name="text">
        <string>Прізвище</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lastnameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="firstname">
       <property name="text">
        <string>Ім'я</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="firstnameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="middlename">
       <property name="text">
        <string>По батькові</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="middlenameLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passport">
       <property name="text">
        <string>Номер паспорта</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="passportLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="telephone">
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="telephoneLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="birthday">
       <property name="text">
        <string>Дата народження</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="birthdayDateEdit">
       <property name="displayFormat">
        <string>yyyy.MM.dd</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
